### PR TITLE
chore(cdk): add textfield-controller migration

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes # 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/cdk/schematics/ng-update/constants/templates.ts [hunk 1]
```diff
@@ -386,6 +386,42 @@ export const ATTRS_TO_REPLACE: ReplacementAttributes[] = [
             attrName: '[tuiDropdown]',
         },
     },
+    {
+        from: {
+            attrName: 'tuiTextfieldExampleText',
+            withTagNames: ['tui-input-card', 'tui-input-tag', 'tui-multiselect'],
```
Comment (nsbarsukov) [inline projects/cdk/schematics/ng-update/constants/templates.ts:392]:
```suggestion
            withTagNames: ['tui-input-card', 'tui-input-tag', 'tui-multi-select'],
```

File: projects/cdk/schematics/ng-update/constants/templates.ts [hunk 2]
```diff
@@ -386,6 +386,42 @@ export const ATTRS_TO_REPLACE: ReplacementAttributes[] = [
...
+            withTagNames: ['tui-input-tag', 'tui-text-area'],
+        },
+        to: {
+            attrName: 'maxLength',
+        },
+    },
+    {
+        from: {
+            attrName: '[tuiTextfieldExampleText]',
+            withTagNames: ['tui-input-card', 'tui-input-tag', 'tui-multiselect'],
```
Comment (nsbarsukov) [inline projects/cdk/schematics/ng-update/constants/templates.ts:410]:
```suggestion
            withTagNames: ['tui-input-card', 'tui-input-tag', 'tui-multi-select'],
```

**Review by nsbarsukov — CHANGES_REQUESTED** [review]

File: projects/cdk/schematics/ng-update/constants/templates.ts (hunk 2 above)
Comment (vladimirpotekhin) [reply projects/cdk/schematics/ng-update/constants/templates.ts]: fixed

**Comment by vladimirpotekhin** [comment]
@nsbarsukov review again, please)

File: projects/cdk/schematics/ng-update/constants/templates.ts [hunk 3]
```diff
@@ -386,6 +386,42 @@ export const ATTRS_TO_REPLACE: ReplacementAttributes[] = [
...
+            withTagNames: ['tui-input-tag', 'tui-text-area'],
+        },
+        to: {
+            attrName: 'maxLength',
+        },
+    },
+    {
+        from: {
+            attrName: '[tuiTextfieldExampleText]',
+            withTagNames: ['tui-input-tag', 'tui-multiselect'],
```
Comment (nsbarsukov) [inline projects/cdk/schematics/ng-update/constants/templates.ts:410]: I've found new one)

```suggestion
            withTagNames: ['tui-input-tag', 'tui-multi-select'],
```

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**